src/vulcan.js

```diff
@@ -5,7 +5,7 @@
 
 var nconf = require('nconf'),
     util = require('./util'),
-    appConfigPath = util.url.resolve(process.cwd(), '/config');
+    appConfigPath = util.url.resolve(process.cwd()+'/', 'config');
```

Comment:
This is what i do not know why about, url.resolve really doesn't help much.... :(